.github/actions/setup-binary-builds/action.yml

```
@@ -108,16 +108,23 @@ runs:
         run: |
           set -euxo pipefail
           CONDA_ENV="${RUNNER_TEMP}/conda_environment_${GITHUB_RUN_ID}"
+          if [[ "${PACKAGE_TYPE:-}" == "conda" ]]; then
+            # For conda package host python version is irrelevant
+            export PYTHON_VERSION=3.9
+            export CONDA_BUILD_EXTRA="conda=23.10.0 conda-build=3.27.0"
+          else
+            # For wheel builds we don't need neither conda nor conda-build
+            export CONDA_BUILD_EXTRA=""
+          fi
 
           conda create \
             --yes --quiet \
             --prefix "${CONDA_ENV}" \
             "python=${PYTHON_VERSION}" \
             cmake=3.22 \
-            conda-build=3.27.0 \
-            conda=23.10.0 \
             ninja=1.10 \
             pkg-config=0.29 \
+            "${CONDA_BUILD_EXTRA}" \
```

```suggestion
            ${CONDA_BUILD_EXTRA} \
```

Since this is the failure for both conda and wheel builds - no quotes needed:
``
+ __conda_exe create --yes --quiet --prefix /__w/_temp/conda_environment_7053546156 python=3.8 cmake=3.22 ninja=1.10 pkg-config=0.29 '' wheel=0.37
``

However we may also need to add lint ignore rule on top of this

---

We don't check this file by lintrunner, so...